ogticrd/covid-safe-paths
# changing news size to adjust the date

#### Description:


I changed the number of lines in the description and the size to fit the date correctly 

#### Linked issues:


Fix [#113](https://trello.com/c/L2j8qCHd/113-fix-the-size-of-the-font-in-news-for-date)

#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/84032847-f2e36000-a965-11ea-9950-198fbd6d8d50.png)


#### How to test:


- `git pull && git checkout Fix/113-size-news-date`
- Head over news screen and check the description of every news block if they fit correctly
